prepared.go

```diff
@@ -0,0 +1,19 @@
+package tarantool
+
+type PreparedID uint64
+
+// Prepared is a type for handling prepared statements
+//
+// Since 1.7.0
```

Comment:
It can be moved into the header of the file.

Reply:
The result is not satisfiable. If I move this note to the top of the file, it gets rendered as a Note for tarantool package.